scripts/install_cmake.sh

```diff
@@ -6,16 +6,18 @@
 
 set -e
 
-VERSION=3.7.1
-PREFIX=~/.local
+VERSION_MAJOR=3
+VERSION_MINOR=15
+VERSION_MICRO=2
+VERSION=$VERSION_MAJOR.$VERSION_MINOR.$VERSION_MICRO
+PREFIX="/usr/local"
```

Comment:
Haha, ok, apparently we were wrong in assuming that this is run as root in travis-ci and apparently whichever user does run it, doesn't have write access to ``/usr/local``, so this still fails...